### PR TITLE
Update latest gradle versions and android dependencies.

### PR DESCRIPTION
**Pull request by IgorGanapolsky**

### CODE REVIEW

File: library/build.gradle [hunk 1]
```diff
@@ -16,13 +16,13 @@ archivesBaseName = 'android-maps-utils'
...
-    compile 'com.google.android.gms:play-services:6.+'
+  compile 'com.google.android.gms:play-services:7.8.0'
 }
 
 
 android {
-    compileSdkVersion 22
-    buildToolsVersion "22.0.1"
+  compileSdkVersion 23
+  buildToolsVersion "23.0.0"
```
Comment (broady) [inline library/build.gradle]: revert indentation in this file

File: demo/build.gradle [hunk 2]
```diff
@@ -14,12 +14,12 @@ dependencies {
...
     // compile 'com.google.maps.android:android-maps-utils:0.3+'
-    compile 'com.google.android.gms:play-services:6.+'
+  compile 'com.google.android.gms:play-services:7.8.0'
 }
 
 android {
-    compileSdkVersion 22
-    buildToolsVersion "22.0.1"
+  compileSdkVersion 23
+  buildToolsVersion "23.0.0"
```
Comment (broady) [inline demo/build.gradle]: revert indentation

**Comment by IgorGanapolsky** [comment]
I signed it!

File: demo/build.gradle (hunk 2 above)
Comment (IgorGanapolsky) [inline demo/build.gradle]: reverted

Comment (broady) [inline demo/build.gradle]: for `build.gradle` and `demo/build.gradle`, too, please.

Comment (IgorGanapolsky) [inline demo/build.gradle]: done...

**Comment by broady** [comment]
Cool, thanks. It looks like you haven't signed the Google CLA, but you signed the Android CLA. Can you check, please?

Reply with `I signed it!` again afterward.

**Comment by IgorGanapolsky** [comment]
I signed it!

**Pull request merged**